code/modules/projectiles/guns/projectile/pistol/olivaw.dm

```diff
@@ -6,11 +6,10 @@
 	item_state = "pistol"
 	fire_sound = 'sound/weapons/guns/fire/pistol_fire.ogg'
 	caliber = CAL_PISTOL
-	ammo_mag = "mag_lpistol"
 	origin_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 3)
 	can_dual = 1
 	load_method = MAGAZINE
-	mag_well = MAG_WELL_PISTOL
```

Comment:
How is that related to armor PR?

Reply:
It is not, I just dont know how to remove it since im not experienced with github at all. If you'd be so kind to tell me how to remove it, I'll do it.